.github/workflows/ondemand-pr-build.yaml

```diff
@@ -0,0 +1,64 @@
+name: on-demand build the copr way
+on:
+  pull_request:
+    types: [labeled]
```

Comment:
We have OST trigger based on comment (/ost). Would it make sense here too? No need to delabel later

Reply:
I started as a comment processing but I couldn't get the action to run in the PR.  Swapping to a comment would be preferred for sure.  I'll try that out.

Reply:
Using a comment based action now!

Reply:
**Note**: Actions defined for `on: issue_comment` that only exist in a (remote?) PR branch are not available to run on the PR itself.  Those actions must be committed to the main/master branch to be available.